maintenance/importConfig.ts

```diff
@@ -245,6 +246,21 @@ async function cleanTmpDirectory(): Promise<void> {
 	console.log("temporary directory cleaned");
 }
 
+function matchId(
+	manufacturer: string,
+	prodType: string,
+	prodId: string,
+): boolean {
+	return (
+		program.ids !== undefined &&
```

Comment:
Now this line isn't necessary anymore 😬

Reply:
Maybe that caused your confusion :D

Reply:
Oh yep sorry :smile:

Reply:
Now I know why I added that, because the return needs to be bool. But now I have fixed by adding `!!`